__tests__/app.test.js

```diff
@@ -15,3 +26,35 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of objects containing the information ", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+        expect(topics[0]).toMatchObject({
```

Comment:
I think this second expect statement is unnecessary here, as we're already checking each object in the array matches the expected structure in the one above.